release-team/role-handbooks/communications/README.md

```diff
@@ -84,7 +84,7 @@ This is a general example of what a release cycle might look like and the order
         <li>The current release cycle's Communications Lead & Comms Shadows
         <li>SIG Release Chairs 
         </ul></ul>
-        <p><a href="https://git.k8s.io/k8s.io/groups/groups.yaml">Membership</a> for this group is defined in <a href="https://git.k8s.io/k8s.io">kubernetes/k8s.io</a>.</p>
+        <p>[Membership](https://git.k8s.io/k8s.io/groups/groups.yaml) for this group is defined in [kubernetes/k8s.io](https://git.k8s.io/k8s.io)</p>
```

Comment:
This will not render well, because there is already HTML around:
![screenshot](https://user-images.githubusercontent.com/695473/104697317-dd11f680-570f-11eb-84d4-c4a0ece33f41.png)

Reply:
@saschagrunert Has been done. PTAL & let me know if there are any other changes required.